ufo2fontir/src/source.rs

```diff
@@ -939,15 +1008,15 @@ impl Work<Context, WorkError> for KerningWork {
                         return None;
                     }
                     let group_name = GroupName::from(name);
-                    if !kerning.groups.contains_key(&group_name) {
-                        debug!("'{name}' references non-existent group {group_name}; ignored");
+                    let Some(group_name) = old_to_new_group_name.get(&group_name) else {
+                        debug!("'{name}' is not a valid group name; ignored");
                         return None;
-                    }
-                    Some(KernParticipant::Group(group_name))
+                    };
+                    Some(KernParticipant::Group((**group_name).clone()))
                 } else {
                     let glyph_name = GlyphName::from(name);
                     if !static_metadata.glyph_order.contains(&glyph_name) {
-                        debug!("'{name}' references non-existent glyph {glyph_name}; ignored");
+                        debug!("'{name}' is not a valid glyph name; ignored");
```

Comment:
again, warning would be better

the message should make it clear that the glyph name is not simply "invalid", but the glyph referenced in the kerning does't exist in the font

Reply:
Done